js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,90 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone( );
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+  }
+
+  get events() {
+    return {
+      'change input[name="windowControlStyle"]': 'changedWindowStyle'
+    };
+  }
+
+  changedWindowStyle( event ) {
+    if ( event.target.id == 'windowControlStyleMac' ) 
+      app.localSettings.set('windowControlStyle', "mac" ); 
+    else
+      app.localSettings.set('windowControlStyle', "win" ); 
+  }
+
+  getFormData( subset = this.$formFields ) {
+    return super.getFormData( subset );
+  }
+
+  saveServer() {
+    const formData = this.getFormData();
+
+    this.settings.set(formData);
+
+    const save = this.settings.save();
+
+    this.trigger('saving');
+
+    if (!save) {
+      // client side validation failed
+      this.trigger('saveComplete', true);
+    } else {
+      this.trigger('savingToServer');
+
+      save
+        .done(() => {
+          this.trigger('saveComplete');
+        })
+        .fail((...args) => {
+          this.trigger('saveComplete', false, true,
+            args[0] && args[0].responseJSON && args[0].responseJSON.reason || '');
+        });
+    }
+
+    this.render();
+    const $firstErr = this.$('.errorList:first');
+    if ($firstErr.length) $firstErr[0].scrollIntoViewIfNeeded();
+  }
+
+  saveLocal() {
+    const localData = this.getFormData(this.$localFields);
+    app.localSettings.set( localData );
+  }
+
+  save() {
```

Comment:
As of now this is never being called and localSettings (except Window Style) are not being saved at all.

I also think this method needs to be smarter. For example, I would only call save if both the localSettings and this.settings (server settings) models passed client validation. If either failed client side validation, I would let the user fix the errors first. It would be odd to me to have a local setting fail validation, we display a red error message, but server settings passed client validation and we show the status bar with "Saved!".